CHANGELOG.md

```diff
@@ -21,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 * Fixed support for `compas_gpython` in Rhino 8 Grasshopper CPython components.
+* Changed installation instructions for Rhino 8 in the user guide.
```

Comment:
ups? sneaked in from another pr?

Reply:
i merged main into this and therefore it got added